benchmarks/analyzeNode/analyze_node_test.go

```diff
@@ -0,0 +1,33 @@
+package detectconent
+
+import (
+	"fmt"
+	"runtime"
+	"testing"
+
+	node "github.com/stackrox/scanner/pkg/analyzer/nodes"
+	"github.com/stretchr/testify/require"
+
+	// Register the Docker image extractor
+	_ "github.com/stackrox/scanner/ext/imagefmt/docker"
+)
+
+func BenchmarkAnalyzeNode(b *testing.B) {
+	runBenchmarkAnalyzeNode(b, "fake path")
+}
+
+func runBenchmarkAnalyzeNode(b *testing.B, pathName string) {
+	var m1, m2 runtime.MemStats
+	b.ResetTimer()
+	runtime.GC()
+	runtime.ReadMemStats(&m1)
+
+	for i := 0; i < b.N; i++ {
+		var err error
+		_, err = node.Analyze("testNode", pathName, node.AnalyzeOpts{UncertifiedRHEL: false, IsRHCOSRequired: true})
```

Comment:
We can ignore the error completely here - checking the error costs us additional time that will be included in the stats. We want to have only `node.Analyze` inside the loop.